syftlib/src/main/java/org/openmined/syft/WebRTC.kt

```diff
@@ -85,8 +95,23 @@ class WebRTCClient(
         }
     }
 
-    private fun sendInternalMessage(type: String, message: String?, to: String) {
-        //TODO implement this
+    /**
+     * Send the message via PyGrid to set up connection
+     * @param type
+     * @param message
+     */
+    private fun sendInternalMessage(type: String, message: String, target: String) {
+        if (target != workerId) {
+            Log.d(TAG, "Sending Internal WebRTC message via PyGrid")
+            val jsonMessage = json {
```

Comment:
Could this be a data class?

Reply:
anonymous data class right? since it isnt used anywhere

Reply:
I was thinking it more to avoid the use of strings but it's ok. You are right it is not going to be used anywhere else.